pyodide-build/pyodide_build/tests/test_buildall.py

```diff
@@ -22,12 +23,18 @@ def test_generate_dependency_graph():
     assert pkg_map["beautifulsoup4"].dependents == set()
 
 
-def test_generate_packages_json():
+def test_generate_packages_json(tmp_path):
+    # Set up directory to store dummy package files for SHA-256 hash verification
+    if not os.path.exists(tmp_path):
+        os.makedirs(tmp_path)
     pkg_map = buildall.generate_dependency_graph(PACKAGES_DIR, {"pkg_1", "pkg_2"})
     for pkg in pkg_map.values():
         pkg.file_name = pkg.file_name or pkg.name + ".file"
+        # Write dummy package file for SHA-256 hash verification
+        with open(os.path.join(tmp_path, pkg.file_name), "w") as f:
```

Comment:
```suggestion
        with open(tmp_path / pkg.file_name, "w") as f:
```

Reply:
Did not know this syntactic sugar exists! Sweet.